app/main.development.js

```diff
@@ -179,6 +181,14 @@ const launchDCRWallet = () => {
     args.push('--testnet');
   }
 
+  // Add any extra args if defined.
+  if (argv.extrawalletargs != undefined) {
+    var extraArgs = argv.extrawalletargs.split(' ');
```

Comment:
does this break passing multiple extra flags since they are separated by spaces?